docker/entrypoint

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+
+usermod -u "$USER_ID" outpack
```

Comment:
Yes this was a huge fight lol. The issue with `-u` was that the cron job needs to be executed as the same user, needs a home directory etc, so it has to be a user that exists inside the docker container, although I suppose actually that could be created at this point using `sudo` rather than during docker build. But the env var option seemed ok as well. 

But didn't realise named volumes avoided this issue, that would be much nicer (for the most part, we do reference the outpack directory during testing in OW, but probably some other way to do that test).

Reply:
Ok no I forgot the other reason - the external user doesn't have write permisisons inside the docker container, so can't create the script to execute (or edit the schedule).